README.md

```diff
@@ -92,6 +92,21 @@ pipeline {
 }
 ```
 
+### Detecting whether a build was started by the trigger in a script:
+```groovy
+def triggerCause = currentBuild.rawBuild.getCause(
+    org.jenkinsci.plugins.pipeline.github.trigger.IssueCommentCause
+)
+
+if (triggerCause.asBoolean()) {
+    echo("Build was started by ${triggerCause.getUserLogin()}, who wrote: " +
```

Comment:
Can be `${triggerCause.userLogin}`, `${triggerCause.comment}` and `${triggerCause.triggerPattern}`

Reply:
Right. Still getting familiar w/ Groovy.